Gemfile.lock

```diff
@@ -421,41 +423,37 @@ GEM
       sprockets (>= 2.8, < 4.0)
       sprockets-rails (>= 2.0, < 4.0)
       tilt (>= 1.1, < 3)
-    selenium-webdriver (2.53.4)
+    selenium-webdriver (3.0.3)
```

Comment:
Lets be narrower with webdriver, capybara, etc updates vs flakey CI (upgrade in individual commits so we can pinpoint regressions)